ui/app/common/domain/services/configurationService.js

```diff
@@ -195,6 +195,18 @@ angular.module('bahmni.common.domain')
             });
         };
 
+        configurationFunctions.prescriptionEmailToggle = function () {
```

Comment:
@kavitha-sundararajan what does prescriptionEmailToggle holds and how are we using this to drive the functionality here ?

Reply:
@binduak prescriptionEmailToggle is a global property for on and off of email functionality. Only if it is set as true, the email icon will be visible in the share options UI. By default it will be false and the icon won't be visible under share options.

Reply:
why it can't be a configuration ? we are validating this value on both UI/backend ?only on UI ?

Reply:
We are validation this value on UI only. Long back, Gurpreet suggested this to be a global property instead of configuration because if we kept it as config, then it would require re-deployment of clinic-config and this should be changeable in the UI itself.

Reply:
no @kavitha-sundararajan i am still not convinced as this property is not being used in the backend. And this is still can be a config parameter.

Reply:
in whatever case, backend must check whether the property is enabled to send email and do so accordingly.

Reply:
@kavitha-sundararajan yes..if we are checking this on backend then it makes sense having it as a global property

Reply:
@kavitha-sundararajan are you checking against that that on the backend anyway?

Reply:
@binduak @angshu the validation is not there previously. have updated bahmni-core with the property check before sending email and raised [PR](https://github.com/Bahmni/bahmni-core/pull/194)